### PR TITLE
Clean up use of base-url and auth-service in command examples

### PR DESCRIPTION
**Pull request by halkyon**
base-url and auth-service already have defaults, so we don't need to use them in the example commands and it will be less confusing to the user.

### CODE REVIEW

File: app/dcs/api/uplink-cli/access-command/access-register/page.md [hunk 1]
```diff
@@ -27,11 +27,11 @@ uplink access register <flags> <ACCESS-GRANT>
...
 
-| Flag                    | Description                                                      |
-| :---------------------- | :--------------------------------------------------------------- |
-| `--access string`       | the serialized access, or name of the access to use              |
-| `--auth-service string` | the address to the service you wish to register your access with |
-| `--public`              | if the access should be public. Default false                    |
+| Flag                    | Description                                                                                          |
+| :---------------------- | :--------------------------------------------------------------------------------------------------- |
+| `--access string`       | the serialized access, or name of the access to use                                                  |
+| `--auth-service string` | the address to the service you wish to register your access with (default "auth.storjshare.io:7777") |
```
Comment (ferristocrat) [inline app/dcs/api/uplink-cli/access-command/access-register/page.md:32]: Is the port actually defined in the default?

File: app/dcs/api/uplink-cli/share-command/page.md [hunk 2]
```diff
@@ -169,15 +169,15 @@ You can also generate a URL to share your projects/buckets/objects
 {% code-group %}
 
 ```windows
-./uplink.exe share sj://cakes/ --url --not-after=none --base-url=https://link.storjshare.io
+./uplink.exe share sj://cakes/ --url --not-after=none
```
Comment (ferristocrat) [inline app/dcs/api/uplink-cli/share-command/page.md:172]: Wondering if we should add `--not-after=none` to the custom domain/tls docs... it'll error if this flag isn't set, right?

File: app/dcs/code/static-site-hosting/page.md [hunk 3]
```diff
@@ -19,32 +19,32 @@ You can use your own domain name and host your own static website on Storj
...
-2.  Share the bucket or object prefix (not individual objects) that will be the root of your website/subdomain. At the root, name your home page `index.html`. The website will serve the index.html file automatically e.g. `http://www.example.test` and `http://www.example.test/index.html` will serve the same content. Anything shared with `--dns` will be _readonly_ and available _publicly_ (no secret key needed).
+2. Share the bucket or object prefix (not individual objects) that will be the root of your website/subdomain. At the root, name your home page `index.html`. The website will serve the index.html file automatically e.g. `http://www.example.test` and `http://www.example.test/index.html` will serve the same content. Anything shared with `--dns` will be _readonly_ and available _publicly_ (no secret key needed).
 3. Finally, you can optionally add the `--tls` flag in order to return an additional DNS entry used for securing your domain with TLS.
 
 {% tabs %}
 {% tab label="Windows" %}
 
 ```Text
-./uplink.exe share --dns <hostname> sj://<bucket>/<prefix> --base-url <linkshare url> --tls
+./uplink.exe share --dns <hostname> sj://<bucket>/<prefix> --tls
```
Comment (ferristocrat) [inline app/dcs/code/static-site-hosting/page.md:31]:
```suggestion
./uplink.exe share --dns <hostname> sj://<bucket>/<prefix> --tls --not-after=none
```

File: app/dcs/code/static-site-hosting/page.md [hunk 4]
```diff
@@ -19,32 +19,32 @@ You can use your own domain name and host your own static website on Storj
...
 ```
 
 {% /tab %}
 
 {% tab label="Linux" %}
 
 ```Text
-uplink share --dns <hostname> sj://<bucket>/<prefix> --base-url <linkshare url> --tls
+uplink share --dns <hostname> sj://<bucket>/<prefix> --tls
```
Comment (ferristocrat) [inline app/dcs/code/static-site-hosting/page.md:39]:
```suggestion
uplink share --dns <hostname> sj://<bucket>/<prefix> --tls --not-after=none
```

File: app/dcs/code/static-site-hosting/page.md [hunk 5]
```diff
@@ -19,32 +19,32 @@ You can use your own domain name and host your own static website on Storj
...
 ```
 
 {% /tab %}
 
 {% tab label="macOS" %}
 
 ```Text
-uplink share --dns <hostname> sj://<bucket>/<prefix> --base-url <linkshare url> --tls
+uplink share --dns <hostname> sj://<bucket>/<prefix> --tls
```
Comment (ferristocrat) [inline app/dcs/code/static-site-hosting/page.md:47]:
```suggestion
uplink share --dns <hostname> sj://<bucket>/<prefix> --tls --not-after=none
```

**Review by ferristocrat — CHANGES_REQUESTED** [review]

File: app/dcs/api/uplink-cli/access-command/access-register/page.md (hunk 1 above)
Comment (halkyon) [reply app/dcs/api/uplink-cli/access-command/access-register/page.md:32]: yeah, that's the DRPC endpoint this particular command uses. I'm not sure why it's not using https, but if you run `uplink access register -h` you'll see the default used

File: app/dcs/api/uplink-cli/share-command/page.md (hunk 2 above)
Comment (halkyon) [reply app/dcs/api/uplink-cli/share-command/page.md:172]: yeah, I think it will. I'll add that in

**Pull request merged**